client/src/components/NavBar/index.js

```diff
@@ -7,31 +7,63 @@ import ControlPointIcon from '@material-ui/icons/ControlPoint';
 import MoreHorizIcon from '@material-ui/icons/MoreHoriz';
 import { Link } from 'react-router-dom';
 import useStyles from './style';
+import Popover from '../Popover';
+import { context } from '../userProvider';
 
 function NavBar() {
   const classes = useStyles();
+  const isAuthenticated = useContext(context);
+  const [isClickedPlusIcon, setIsClicked] = useState(false);
+
+  const handelClick = () => {
```

Comment:
It might help in the future to give this a more descriptive name, like `handleClickPlusIcon` for example.  In the future, the NavBar might have another clickable button added to it, so you'll need to distinguish which button is being clicked.